src/ansible/roles/common/tasks/main.yml

```diff
@@ -64,6 +64,19 @@
     system: yes
     state: present
 
+- name: 'Check if user with uid {{ user.regular.uid }} already exists'
```

Comment:
Please do not do this, it is bound to break VMs in cloud as the account that is provisioned as default is the only one accessible with sudo priviledges.

Reply:
This is for containers. As far as I know you are using different uid value on vms.

Reply:
https://github.com/SSSD/sssd-ci-containers/pull/21/commits/5daa9e7e9375607be15668c92db16f4ea33f4f66#diff-39d9c8858e131ac0b5ac9afffe35f2a1c4f31f5a49536b15ae338c0660ca5671R12

So what happens is that it will make sure that uid 1024 is not present in the system before creating the user.

Reply:
And since uid 1024 does not exist on the virtual machine, it is a noop.

Reply:
I would rather see changing the ci user.regular.uid on collision so we can get rid of the override in first place.
Removing account to replace it with a new one with the same uid is generally considered as a bad practice.

At least please make sure that the ci user is not re-created so it can figure out that the user it found is the same one created by the playbook (on a re-run).

You closed my other MR that was touching/fixing the uids on ubuntu rolling completely ignoring the second fix that was addressing pip issue. https://github.com/SSSD/sssd-ci-containers/pull/42/files#diff-0fb17d023b75aba7561865659127745cb46523c950aee204c80b8601928561d6R200

Reply:
> I would rather see changing the ci user.regular.uid on collision so we can get rid of the override in first place. Removing account to replace it with a new one with the same uid is generally considered as a bad practice.
> 
> At least please make sure that the ci user is not re-created so it can figure out that the user it found is the same one created by the playbook (on a re-run).
> 
> You closed my other MR that was touching/fixing the uids on ubuntu rolling completely ignoring the second fix that was addressing pip issue. https://github.com/SSSD/sssd-ci-containers/pull/42/files#diff-0fb17d023b75aba7561865659127745cb46523c950aee204c80b8601928561d6R200

This is fixed as well, by installing the package via apt.

Reply:
Well clearly it is not solved in this MR.
```
  - name: Install additional python packages
    pip:
      name:
      - flaky
      - python3-flaky
  when: "'client_devel' in group_names"
  ```

Reply:
Nevertheless the playbook will delete and re-create needlessly the ci user on rerun.
```
getent passwd {{ user.regular.uid }} | cut -d ':' -f 1 | grep -v '^{{ user.regular.name }}$'
```

Reply:
> Well clearly it is not solved in this MR.
> 
> ```
>   - name: Install additional python packages
>     pip:
>       name:
>       - python3-flaky
>   when: "'client_devel' in group_names"
> ```

Because this PR just cherry-picked commits from my work.

Reply:
This is the one: https://github.com/SSSD/sssd-ci-containers/pull/47 too bad you didn't comment there.

Reply:
> Nevertheless the playbook will delete and re-create needlessly the ci user on rerun.
> 
> ```
> getent passwd {{ user.regular.uid }} | cut -d ':' -f 1 | grep -v '^{{ user.regular.name }}$'
> ```

https://github.com/SSSD/sssd-ci-containers/pull/50